commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+git config --global user.name 'github action'
+
+if [ -n "$(git status --porcelain)" ]; then
+	echo "there are changes";
+	git add package/*
+	git commit -m 'publish wheel by github action'
+	git push
```

Comment:
hey @stephanie-mak , does this avoid the infinite loop, where the commit and push would trigger github action to run again, which will then make another build/commit/push?

Reply:
there might be a lock in github action since infinite loop behaviour is not observed in the data asset repo